financial-templates-lib/logger/SlackTransport.js

```diff
@@ -60,6 +60,13 @@ const slackFormatter = info => {
           formattedResponse.blocks[
             formattedResponse.blocks.length - 1
           ].text.text += `    - _tx_: <https://etherscan.io/tx/${info[key][subKey]}|${info[key][subKey]}> \n`;
+        }
+        // If the value within the object itself is an object we dont want to spread it any further. Rather,
+        // convert the object to a string and print it along side it's key value pair.
+        if (typeof info[key][subKey] === "object" && info[key][subKey] !== null) {
+          formattedResponse.blocks[
+            formattedResponse.blocks.length - 1
+          ].text.text += `    - _${subKey}_: ${JSON.stringify(info[key][subKey])}\n`;
```

Comment:
You might want to make the block of spaces, that you use to indent, a constant so that we can enforce consistency. For example you could add a `const ident = "    "`

Reply:
Good idea, and then you can use the `indent.repeat(n)` to specify the level of indent.